src/ray/core_worker/reference_count_test.cc

```diff
@@ -16,6 +19,124 @@ class ReferenceCountTest : public ::testing::Test {
   virtual void TearDown() {}
 };
 
+class MockWorkerClient : public rpc::CoreWorkerClientInterface {
+ public:
+  MockWorkerClient(ReferenceCounter &rc, const std::string &addr)
+      : rc_(rc), task_id_(TaskID::ForFakeTask()) {
+    address_.set_ip_address(addr);
+    address_.set_raylet_id(ClientID::FromRandom().Binary());
+    address_.set_worker_id(WorkerID::FromRandom().Binary());
+  }
+
+  // TODO: Make message receive and reply async.
```

Comment:
Can you leave your name and also expand this a bit? Don't know what it means w/o more context